### PR TITLE
fix(evpn): unit test fixes

### PR DESCRIPTION
**Pull request by atulpatel261194**

### CODE REVIEW

File: network/logical_bridge_test.go [hunk 1]
```diff
@@ -3,161 +3,220 @@
 // Copyright (c) 2022-2023 Dell Inc, or its subsidiaries.
 
 // Package network implements the go library for OPI to be used to establish networking
-package network_test
+package network
 
 import (
 	"context"
-	"errors"
+	// "errors"
```
Comment (artek-koltun) [inline network/logical_bridge_test.go:10]: we can just remove the package if it is not used anymore

File: network/logical_bridge_test.go [hunk 2]
```diff
@@ -3,161 +3,220 @@
...
-
-	resultResponse, err := mockEvpnClient.DeleteLogicalBridge(context.TODO(), "lb1", true)
-
+// TestDeleteLogicalBridge tests the DeleteLogicalBridge method
+func TestDeleteLogicalBridge(t *testing.T) {
+	wantRequest := &pb.DeleteLogicalBridgeRequest{
+		Name:         "//network.opiproject.org/bridges/lb1",
+		AllowMissing: true,
+	}
+	wantResponse := &emptypb.Empty{}
```
Comment (artek-koltun) [inline network/logical_bridge_test.go:83]: we don't really need to test expected response for `Empty` type. It is just an empty message, which cannot be used in any way

File: network/vrf_test.go [hunk 3]
```diff
@@ -3,160 +3,383 @@
...
 	"context"
 	"errors"
 	"testing"
 
 	"github.com/opiproject/godpu/mocks"
-	_go "github.com/opiproject/opi-api/network/evpn-gw/v1alpha1/gen/go"
+	pb "github.com/opiproject/opi-api/network/evpn-gw/v1alpha1/gen/go"
+	"google.golang.org/grpc"
+
+	// pc "github.com/opiproject/opi-api/network/opinetcommon/v1alpha1/gen/go"
```
Comment (artek-koltun) [inline network/vrf_test.go:17]: this line can be removed

File: network/logical_bridge_test.go (hunk 1 above)
Comment (artek-koltun) [inline network/logical_bridge_test.go:10]: can be removed

**Review by artek-koltun — COMMENTED** [review]
@atulpatel261194 a great improvements of the tests! Nice to see +17.80% of the overall coverage. However, there are some comments from my side.

File: network/bridge_port_test.go [hunk 4]
```diff
@@ -3,162 +3,392 @@
...
-	assert.Equal(t, expectedBridgePort, resultBridgePort)
-	mockEvpnClient.AssertExpectations(t)
+	assert.True(t, proto.Equal(response, wantResponse))
+	assert.Equal(t, true, connClosed)
 }
 
+// TestCreateBridgePortError tests the CreateBridgePort method
 func TestCreateBridgePortError(t *testing.T) {
-	mockEvpnClient := &mocks.EvpnClient{}
+	wantRequest := &pb.CreateBridgePortRequest{
```
[the remark this answers is not in the document]
Comment (atulpatel261194) [reply network/bridge_port_test.go:69]: @artek-koltun I appreciate your feedback, I want to acknowledge that I am currently on leave for the next two weeks. However, please be assured that I am committed to addressing the suggestions you provided as soon as I return in January'24 . I have made note of your suggestions and will prioritize implementing the necessary changes to improve the quality and effectivenes.

**Comment by atulpatel261194** [comment]
> @atulpatel261194 a great improvements of the tests! Nice to see +17.80% of the overall coverage. However, there are some comments from my side.

I appreciate your feedback, I want to acknowledge that I am currently on leave for the next two weeks. However, please be assured that I am committed to addressing the suggestions you provided as soon as I return in January'24 . I have made note of your suggestions and will prioritize implementing the necessary changes to improve the quality and effectivenes

**Comment by artek-koltun** [comment]
> I appreciate your feedback, I want to acknowledge that I am currently on leave for the next two weeks. However, please be assured that I am committed to addressing the suggestions you provided as soon as I return in January'24 . I have made note of your suggestions and will prioritize implementing the necessary changes to improve the quality and effectivenes

Sure. Enjoy your vacation!

File: network/svi.go [hunk 5]
```diff
@@ -122,12 +122,9 @@ func (c evpnClientImpl) UpdateSvi(ctx context.Context, name string, updateMask [
 	}
 	defer closer()
 	client := c.getEvpnSVIClient(conn)
-	svi, err := client.GetSvi(ctx, &pb.GetSviRequest{
+
```
Comment (artek-koltun) [inline network/svi.go:125]: can we extract that change in `svi.go` file into a separate commit like `fix(network/svi): create svi directly for update`

**Review by artek-koltun — COMMENTED** [review]
Looks good. Just some minor comments

**Review by artek-koltun — APPROVED** [review]
LGTM!
